src/notation/view/widgets/editstyle.ui

```diff
@@ -6288,6 +6288,13 @@
              </layout>
             </widget>
            </item>
+           <item row="0" column="2">
+            <widget class="QToolButton" name="resetMinEmptyMeasures">
+             <property name="text">
+              <string>...</string>
+             </property>
+            </widget>
+           </item>
```

Comment:
That is not enough. Check my PR for Mu3, https://github.com/Jojo-Schmitz/MuseScore/pull/578

Reply:
Has musescore.qrc moved since 3.x? I can't seem to find it. On this branch (master) it looks like the icon is set in `editstyle.cpp` line 986.

Reply:
May well be. But yiu need the toolTip, the empty and non-trananslatable text and the accessibilityName

Reply:
Icons are indeed handled very differently nowadays; those are added via editstyle.cpp now. But indeed, the other three things are all still relevant.